### PR TITLE
train

### PR DESCRIPTION
**Pull request by MattLui**
Made some simple changes, see the edits to the readme file to see what has been updated in the "train" version of our app!

### CODE REVIEW

File: public/index.html
Comment (timw-csi) [inline public/index.html:1]: this works for main landing page, but line 7 of `views/binView.ejs` would need to be changed for diaeresis to be removed from the /{binId}/view page as well.

Comment (MattLui) [reply public/index.html:1]: Nice catch, thanks! I will push change and create new PR

Comment (MattLui) [reply public/index.html:1]: Actually it looks like by committing and pushing it automatically updated this PR, so you should be able to see and approve this one

Comment (timw-csi) [reply public/index.html:1]: oh nice! that makes the process a bit easier

**Review by timw-csi — APPROVED** [review]

**Pull request merged**